src/Tizen.NUI.Components/Controls/Tab.cs

```diff
@@ -145,8 +133,6 @@ public int ItemGap
         /// Left space in Tab.
         /// </summary>
         /// <since_tizen> 6 </since_tizen>
-        /// This will be public opened in tizen_5.5 after ACR done. Before ACR, need to be hidden as inhouse API.
-        [EditorBrowsable(EditorBrowsableState.Never)]
         public int LeftSpace
```

Comment:
Why don't you use just Padding of View? If it is possible, please remove the 4 Spaces properties

Reply:
Modify to Extents